src/ResMenuItem.h

```diff
@@ -9,23 +9,25 @@ static inline CFDictionaryRef CGDisplayModeGetDictionary(CGDisplayModeRef mode)
 
 @interface ResMenuItem : NSMenuItem
 {
-	CGDirectDisplayID _display;
-	
-	int modeNum;
-	
-	//CGDisplayModeRef _mode;
-	
-	float refreshRate;
-	float scale;
-	int colorDepth;
-	int width;
-	int height;
+    CGDirectDisplayID _display;
+    
+    int modeNum;
+    
+    //CGDisplayModeRef _mode;
+    
+    int refreshRate;
```

Comment:
MacBook Pro 16 [supports non-integer refresh rates](https://www.provideocoalition.com/apple-macbook-pro-16-supports-non-integer-framerates/).

Reply:
Yes it does, but the retrieved data would always be an integer. Please refer to https://github.com/jnooree/RDM/blob/b6935468139b8a62c73ec822d2d5b60cc2cb47d9/src/utils.h#L32.

Reply:
Then it's interesting how it should work on whe MBP.

Reply:
I'm on that mac, and it fails to display the correct refresh rate. I've tried several thing to make it work again, but unfortunately, none of them worked.